Documentation/ApiOverview/Fluid/UsingFluidInTypo3.rst

```diff
@@ -15,10 +15,58 @@ Here are some examples of how Fluid can be used in TYPO3:
     creation of a sitepackage extension.
 *   :ref:`adding-your-own-content-elements` in addition to the already existing
     content elements TYPO3 supplies.
-*   The previous point describes the lightweight components which
-    are created using a combination of TypoScript and Fluid. If you need more
-    functionality or flexibility in your content element, you can create a
-    content plugin using a combination of Extbase and Fluid.
+*   :ref:`Extbase-based controllers <extbase-controller>` have a default Fluid
+    view in :php:`$this->view`.
 *   Use Fluid to create emails using the :ref:`TYPO3 Mail API <mail-fluid-email>`.
 *   Use Fluid in :ref:`backend modules <backend-modules-template>`, either with or
     without Extbase.
+*   Use the :ref:`generic view factory <generic-view-factory>` to create a
+    Fluid view.
+
+..  deprecated:: 13.3
+    These classes have been marked as deprecated in TYPO3 v13 and will be removed in v14:
+
+    *   :php:`TYPO3\CMS\Fluid\View\StandaloneView`
+    *   :php:`TYPO3\CMS\Fluid\View\TemplateView`
+    *   :php:`TYPO3\CMS\Fluid\View\AbstractTemplateView`
+    *   :php:`TYPO3\CMS\Extbase\Mvc\View\ViewResolverInterface`
+    *   :php:`TYPO3\CMS\Extbase\Mvc\View\GenericViewResolver`
+
+..  _generic-view-factory:
+
+Using the generic view factory (ViewFactoryInterface)
+=====================================================
+
+..  versionadded:: 13.3
+    Class :php:`TYPO3\CMS\Core\View\ViewFactoryInterface` has been added as a
+    generic view factory interface to create views that return an instance of
+    :php:`TYPO3\CMS\Core\View\ViewInterface`. This implements the "V" of "MVC"
+    in a generic way and is used throughout the TYPO3 core.
+
+You can :ref:`inject <dependency-injection>` an instance of the
+:php:`TYPO3\CMS\Core\View\ViewFactoryInterface` to create an instance of a
+:php:`TYPO3\CMS\Core\View\ViewInterface` where you need one.
+
+..  note::
+    :ref:`Extbase-based controllers <extbase-controller>` create a view
+    instance based on this factory by default and are accessible as
+    :php:`$this->view`.
+
+..  literalinclude:: _UsingFluid/_MyController.php
+    :caption: EXT:my_extension/Classes/Controller/MyController.php
```

Comment:
Maybe note here that this is not an extbase example but a custom controller implementation. Might be confusing otherwise because it doesn't use `$this->view` and is `readonly`.

Reply:
I would have argued you can see that from the inheritance but it cannot harm to have an additional hint. Added one